chainerx_cc/chainerx/routines/connection.cc

```diff
@@ -362,8 +362,12 @@ Array Linear(const Array& x, const Array& w, const absl::optional<Array>& b, uin
         }
     }
 
-    BackwardBuilder bb = has_bias ? BackwardBuilder{"linear", {x_matrix, w, b_matrix}, out_matrix}
-                                  : BackwardBuilder{"linear_nobias", {x_matrix, w}, out_matrix};
+    // NOTE: 1. BackwardBuilder's move assignment is deleted.
+    //       2. GCC does not work well with move ctor with ternary move or move ternary as argument.
+    //       3. please refers to issue 8192 and it related PR for more information.
```

Comment:
It looks the most important piece of information, that MSVC fails to compile the original code, is missing.

Also, please write the link as `#8192`.